notebooks/official/pipelines/control_flow_kfp.ipynb

```diff
@@ -530,9 +473,9 @@
       "source": [
         "## Define a pipeline that uses control structures\n",
         "\n",
-        "The following example defines a pipeline that uses these components and demonstrates the use of  `dsl.Condition` and `dsl.ParallelFor`.\n",
+        "The following example, defines a pipeline component that demonstrates the use of `dsl.Condition` and `dsl.ParallelFor`.\n",
```

Comment:
remove the comma after "example". "The following example" is the subject of the sentence, not a parenthetical 👍 

- "The following example defines a pipeline component that..."